domain/consensus/processes/dagtraversalmanager/dagtraversalmanager_test.go

```diff
@@ -3,11 +3,109 @@ package dagtraversalmanager_test
+			currentBlock := highHash
+			currentBlockData, err := tc.GHOSTDAGDataStore().Get(tc.DatabaseContext(), currentBlock)
+			if err != nil {
+				t.Fatalf("GHOSTDAGDataStore().Get: %+v", err)
+			}
+			for i := uint64(0); i <= depth; i++ {
+				if currentBlockData.SelectedParent() == nil {
+					break
+				}
+				currentBlock = currentBlockData.SelectedParent()
+				currentBlockData, err = tc.GHOSTDAGDataStore().Get(tc.DatabaseContext(), currentBlock)
+				if err != nil {
+					t.Fatalf("GHOSTDAGDataStore().Get: %+v", err)
+				}
+			}
+			expected := currentBlock
+			blockHash, err := tc.DAGTraversalManager().BlockAtDepth(highHash, depth)
+			if err != nil {
+				t.Fatalf("BlockAtDepth: %+v", err)
+			}
+
+			if !blockHash.Equal(expected) {
+				t.Fatalf("Expected block %s but got %s", expected, blockHash)
+			}
+		}
+
+		// "checkExpectedBlockTwoChildrenSameSelectedParent" checks blockAtDepth on 2 children with the same selectedParent.
+		checkExpectedBlockTwoChildrenSameSelectedParent := func(firstChild, secondChild *externalapi.DomainHash, depth uint64) {
+			blockHash, err := tc.DAGTraversalManager().BlockAtDepth(firstChild, depth)
+			if err != nil {
+				t.Fatalf("BlockAtDepth: %+v", err)
+			}
+			expectedSameHash, err := tc.DAGTraversalManager().BlockAtDepth(secondChild, depth)
+			if err != nil {
+				t.Fatalf("BlockAtDepth: %+v", err)
+			}
+			if !blockHash.Equal(expectedSameHash) {
+				t.Fatalf("Expected block %s but got %s", expectedSameHash, blockHash)
+			}
+		}
+
+		// "checkExpectedBlockTwoChildrenSameSelectedParent" checks on different chains, on the same DAG, that they merge at the correct point.
+		checkExpectedBlockOnTwoDiffChain := func(firstChild, secondChild *externalapi.DomainHash, firstDepth, secondDepth uint64) {
+			blockHash, err := tc.DAGTraversalManager().BlockAtDepth(firstChild, firstDepth)
+			if err != nil {
+				t.Fatalf("BlockAtDepth: %+v", err)
+			}
+			expectedSameHash, err := tc.DAGTraversalManager().BlockAtDepth(secondChild, secondDepth)
+			if err != nil {
+				t.Fatalf("BlockAtDepth: %+v", err)
+			}
+			if !blockHash.Equal(expectedSameHash) {
+				t.Fatalf("Expected block %s but got %s", expectedSameHash, blockHash)
+			}
+			expectedDiffHash, err := tc.DAGTraversalManager().BlockAtDepth(secondChild, secondDepth-1)
+			if err != nil {
+				t.Fatalf("BlockAtDepth: %+v", err)
+			}
+			if blockHash.Equal(expectedDiffHash) {
+				t.Fatalf("Expected to differente block")
+			}
+
+		}
+
+		highHash, err := createAChainDag(params.GenesisHash, tc)
+		if err != nil {
+			t.Fatalf("createAChainDagInBlockAtDepth: %+v", err)
+		}
+		checkExpectedBlockChainDag(highHash, 2)
+
+		firstChild, secondChild, err := expendDagToTwoChildrenSameSelectedParent(highHash, tc)
+		if err != nil {
+			t.Fatalf("createADagInBlockAtDepth: %+v", err)
+		}
+		checkExpectedBlockTwoChildrenSameSelectedParent(firstChild, secondChild, 2)
+
+		firstChild, secondChild, err = expendDagToDiffChains(firstChild, secondChild, tc)
```

Comment:
Dag (as any other ראשי תיבות) should be written with all words in capital, a.k.a. DAG.
This applies to any other place you wrote Dag in this or any other PR.

Note: This convention is somewhat unique to go. In most other languages Dag would have been the correct way to write.

Reply:
Changed, thanks.